### ISSUE TITLE
 write8 and write16, of webi2c the fourth argument

### ISSUE THREAD

**Issue by MSakamaki** [club-wot/WebGPIO#6]
The fourth argument is 1 byte (octet) flag

https://github.com/dadaa/gecko-dev/blob/b2g44_v2_5-mozopenhard/dom/i2c/I2cService.cpp#L139
-  `navigator.mozI2c.write(this.portNumber, registerNumber, value, true);`

https://github.com/club-wot/WebGPIO/blob/draft-20160125/src/WebI2C/interfaces/I2CSlaveDevice/I2CSlaveDevice.js#L99

**Comment by MSakamaki** [comment]
``` javascript

  write8: function (registerNumber, value) {
    return this._write(registerNumber, value, true);
  },

  write16: function (registerNumber, value) {
    return this._write(registerNumber, value, false);
  },
  // @private 
  _write: function(registerNumber, value, aIsOctet) {
    return new Promise((resolve, reject) => {
      navigator.mozI2c.write(this.portNumber, registerNumber, value, aIsOctet);
      resolve(value);
    });
  }

```

**Comment by MSakamaki** [comment]
``` javascript


  read8: function (readRegistar) {
    return this._read(readRegistar, true);
  },

  read16: function (readRegistar) {
    return this._read(readRegistar, false);
  },

  // @private 
  _read: function (readRegistar, aIsOctet) {
    return new Promise((resolve, reject) => {
      resolve(navigator.mozI2c.read(this.portNumber, readRegistar, aIsOctet));
    });
  },

```

**Comment by MSakamaki** [comment]
Fixed in #21 
thx! @tadfmac
